load_collection.json

```diff
@@ -208,7 +208,7 @@
         }
     ],
     "returns": {
-        "description": "A data cube for further processing.",
+        "description": "A data cube for further processing. The dimensions and dimension properties (name, type, labels, reference system and resolution) correspond to the collection's metadata, but the dimension labels got restricted as specified in the parameters.",
```

Comment:
I think  `labels got restricted` can be just `labels are restricted`

Reply:
`lables are restricted` sounds good to me.